core/lda_engine.py

```diff
@@ -102,8 +101,10 @@ def get_author_top_topics(self, author_id, top=10):
 
         :return: a NumPy array of topics IDs and their confidence levels.
         """
-
-        author = self.authors_lib[author_id]
+        try:
+            author = self.authors_lib[author_id]
```

Comment:
authors_lib.get ?

Reply:
It would be tedious to write in a .get() manner, as you can infer from the context.

Reply:
Will there any chance of really having no such an id in dict?

Reply:
Yes, and that should definitely throw an error and crash the app. It is a critical error as it may imply that the user are not configuring the authors_lib correctly.